pkg/envoy/types.go

```diff
@@ -90,7 +90,7 @@ const (
 // *Note: HTTP typed filters referenced in RDS require a wellknown name
 const (
 	// HTTP filters
-	HTTPConnectionManagerFilterName = "http_connection_manager"
+	HTTPConnectionManagerFilterName = "envoy.filters.network.http_connection_manager"
```

Comment:
Should this reference the wellknown names package?

Reply:
done!